fairseq/models/masked_lm.py

```diff
@@ -229,8 +234,13 @@ def forward(self, src_tokens, segment_labels=None, **unused):
         elif self.embed_out is not None:
             x = self.embed_out(x)
 
+       
```

Comment:
This looks good to me!  Can we remove these extra newlines though?  Thanks!

Reply:
Done!